run.py

```diff
@@ -0,0 +1,7 @@
+import os, subprocess, locale
+cwd = os.getcwd()
+def cmd(execute):
+    return subprocess.call(execute.format(cwd).split())
```

Comment:
Splitting on whitespace is error-prone. Let's use `subprocess.call([arg1, arg2, argN])` instead.